submissions/famuyiwaOlawumi/HTML/index.html

```diff
@@ -0,0 +1,170 @@
+<!DOCTYPE html>
```

Comment:
Well done! You are require to use ONLY 2 product images. Delete the images that you are not using.

Reply:
Can I get more clarification on this? 
The first two images I inserted is for the actual product of interest. The others is for similar products. Are you saying I don't need images to depict the section for similar products the buyer might like? Or am I to give 1 image to the actual product of interest and then 1 to similar products of interest?
Thanks

Reply:
In the instructions for this task, you were asked to use only 2 product images but you have 6 images. Remove the extra images and delete the unused ones. Check [here](https://github.com/Dufuna-CodeCamp/Dufuna-CodeCamp20/issues/7) for the instruction. 
Yes, you don't need images to depict the similar products.